static/app/main.tsx

```diff
@@ -20,7 +20,14 @@ function renderRouter(props: any) {
   );
 }
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      // Prevents queries from being refetched if the data is < 5 seconds old
+      staleTime: 5000,
```

Comment:
I would be careful with turning this on by default and stick to `0` as the default. The reason being is that I think it is going to be unlikely that we are going to find an N here that fits the majority of use-cases + since this is somewhat hidden, we may cause confusion with folks being confused why data is not refetched. 

I'd stick to the least opinionated approach here + imo it is simple enough for everyone to pass a new queryclient if they need to.

Reply:
Changed to default in 8efdf321f66186dcafc750b70e5fc2944f8d4ab6

Reply:
> Since setting staleTime to a positive number will be a pretty common config override, I wonder if we can figure out some way to allow consumers to lengthen it without choosing a random number.

This will likely best be known by the developer working on the feature or it may depend on the feature itself. I would say the best way to increase this or customize it is to pass a separate QueryProvider